### DISCUSSION TITLE
New Tool: RemoteLocator

### DISCUSSION THREAD

**Discussion by bengtmartensson** [probonopd/irdb#63] [Show and tell]
I have just written a new tool, [RemoteLocator](https://github.com/bengtmartensson/RemoteLocator), that can be used to locate, download and convert IR remotes on the internet, for example from IRDB (i.e. the GitHub repository, not the defunct irdb.tk). As presented in the GitHub project, it is a command line program, but it is designed to be integrated into GUI programs like IrScrutinizer ([issue](https://github.com/bengtmartensson/IrScrutinizer/issues/429)). (For a first glance, look at the [generated HTML file](http://harctoolbox.org/downloads/remotelocator.html).)

Using the command line program, a remote can be found, downloaded and converted (supported are IRDB CVS, Pronto Hex (as text) and Girr (XML) formats) within a few seconds.

Remotes from the giant ["JP1 master file"](http://www.hifi-remote.com/forums/dload.php?action=file&file_id=26089) can also be browsed, but unfortunately not imported.

All sort of feedback and suggestions are welcome.

**Comment by bengtmartensson** [comment]
I have succeeded in integrating RemoteLocator in [IrScrutinizer](https://github.com/bengtmartensson/IrScrutinizer) in an [experimental build](https://github.com/bengtmartensson/IrScrutinizer/releases/tag/ci-build), see screen shot below:


![Screenshot from 2021-05-31 20-24-21](https://user-images.githubusercontent.com/11508922/120229803-3ce7e480-c24e-11eb-8a8c-ba3b29f909e2.png)

So now it is possible to GUI-browse the IRDB remotes (together with the Lirc remote data base, my "toy" [GirrLib](https://github.com/bengtmartensson/GirrLib), and the  ["JP1 master file"](http://www.hifi-remote.com/forums/dload.php?action=file&file_id=26089) all in one. 

Feedback, bugreports, etc welcome.

**Comment by probonopd** [comment]
That looks really awesome! IrScrutinizer rocks.
